akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -903,152 +903,251 @@ trait LoggingAdapter {
    * @see [[LoggingAdapter]]
    */
-  def error(cause: Throwable, template: String, arg1: Any, arg2: Any, arg3: Any): Unit = { if (isErrorEnabled) notifyError(cause, format(template, arg1, arg2, arg3)) }
+  def error(cause: Throwable, template: String, arg1: Any, arg2: Any, arg3: Any): Unit = {
+    if (isErrorEnabled) notifyError(cause, format(template, arg1, arg2, arg3))
+  }
   /**
    * Message template with 4 replacement arguments.
    * @see [[LoggingAdapter]]
    */
-  def error(cause: Throwable, template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = { if (isErrorEnabled) notifyError(cause, format(template, arg1, arg2, arg3, arg4)) }
+  def error(cause: Throwable, template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = {
+    if (isErrorEnabled) notifyError(cause, format(template, arg1, arg2, arg3, arg4))
+  }
 
   /**
    * Log message at error level, without providing the exception that caused the error.
    * @see [[LoggingAdapter]]
    */
-  def error(message: String): Unit = { if (isErrorEnabled) notifyError(message) }
+  def error(message: String): Unit = {
+    if (isErrorEnabled) notifyError(message)
+  }
   /**
    * Message template with 1 replacement argument.
    * @see [[LoggingAdapter]]
    */
-  def error(template: String, arg1: Any): Unit = { if (isErrorEnabled) notifyError(format1(template, arg1)) }
+  def error(template: String, arg1: Any): Unit = {
+    if (isErrorEnabled) notifyError(format1(template, arg1))
+  }
   /**
    * Message template with 2 replacement arguments.
    * @see [[LoggingAdapter]]
    */
-  def error(template: String, arg1: Any, arg2: Any): Unit = { if (isErrorEnabled) notifyError(format(template, arg1, arg2)) }
+  def error(template: String, arg1: Any, arg2: Any): Unit = {
+    if (isErrorEnabled) notifyError(format(template, arg1, arg2))
+  }
   /**
    * Message template with 3 replacement arguments.
    * @see [[LoggingAdapter]]
    */
-  def error(template: String, arg1: Any, arg2: Any, arg3: Any): Unit = { if (isErrorEnabled) notifyError(format(template, arg1, arg2, arg3)) }
+  def error(template: String, arg1: Any, arg2: Any, arg3: Any): Unit = {
+    if (isErrorEnabled) notifyError(format(template, arg1, arg2, arg3))
+  }
   /**
    * Message template with 4 replacement arguments.
    * @see [[LoggingAdapter]]
    */
-  def error(template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = { if (isErrorEnabled) notifyError(format(template, arg1, arg2, arg3, arg4)) }
+  def error(template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = {
+    if (isErrorEnabled) notifyError(format(template, arg1, arg2, arg3, arg4))
+  }
+  /**
+   * Message template with varargs replacement arguments.
+   * @see [[LoggingAdapter]]
+   */
+  @annotation.varargs
+  def error(template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any, args: Any*): Unit = {
+    if (isErrorEnabled) notifyError(format(template, Seq(arg1, arg2, arg3, arg4) ++ args: _*))
```

Comment:
Please manually allocate the right array size and use `System.arrayCopy` to populate it from `args` (using `args.asInstanceOf[WrappedArray[Any]].array`).